lms/djangoapps/certificates/queue.py

```diff
@@ -174,7 +174,7 @@ def del_cert(self, student, course_id):
 
         raise NotImplementedError
 
-    # pylint: disable=too-many-statements
+    # pylint: disable=too-many-statements, unused-argument
```

Comment:
This is temporary. It will be cleaned up in a separate PR as it will touch a number of files.